packages/mdc-grid-list/foundation.js

```diff
@@ -25,6 +25,7 @@ export default class MDCGridListFoundation extends MDCFoundation {
   static get defaultAdapter() {
     return {
       getOffsetWidth: () => /* number */ 0,
+      isEmpty: () => false,
```

Comment:
I feel like this may be a _bit_ too abstract for an adapter method. Instead, how do you feel about a `getNumberOfTiles()` method?

```js
getNumberOfTiles: () => /* number */ 0,
```

To me this narrows the scope of what the adapter needs to do down to just returning the number of elements the list contains, which needs to be done fully within the host environment. Whereas checking that that number is equal to `0` is something we could do in the foundation.